client/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+
+import './Card.scss'
+import { CardType } from '../../types/CardType'
+
+const cardToEmoji = (card: CardType) => {
+  let emoji
+  let label
+  switch (card) {
+    case CardType.EGG:
+      emoji = '🥚'
+      label = 'Egg'
+      break
+    case CardType.CHICKEN:
+      emoji = '🐔'
+      label = 'Chicken'
+      break
+    case CardType.FOX:
+      emoji = '🦊'
+      label = 'Fox'
+      break
+  }
+  return (
+    <span className='Card__Emoji' role='img' aria-label={label}>
+      {emoji}
```

Comment:
What about using svg images for the post-MVP era instead? Some devices/OS maybe don't render new emojis like 🦊.

Reply:
ofc